part-spec/passives.json

```diff
@@ -0,0 +1,237 @@
+                    "description": "common component identifying information, such as mpn",
+                    "$ref": "definitions.json#/componentID"
+                },
+                "pins": {
+                    "description": "array of pin objects with associated properties",
+                    "type": "array",
+                    "items": {
+                        "$ref": "definitions.json#/pinSpec"
+                    }
+                },
+                "value": {
+                    "description": "resistor value",
+                    "comment": "units of ohms",
+                    "$ref": "definitions.json#/unit"
+                },
+                "tolerance": {
+                    "description": "nominal tolerance of a resistor",
+                    "comment": "units of %",
+                    "$ref": "definitions.json#/unit"
+                },
+                "powerRating": {
+                    "description": "measure of power a resistor can dissipate indefinitely without degrading performance",
+                    "comment": "units of watts",
+                    "$ref": "definitions.json#/unit"
+                },
+                "temperatureCoefficient": {
+                    "description": "change in resistance when the temperature is changed",
+                    "comment": "units of ohms/degree (Celsius or Kelvin)",
+                    "type": "number"
+                },
+                "maxOverloadVoltage": {
+                    "description": "maximum voltage that can be applied to the resistor for a short period of time",
+                    "comment": "units of volts",
+                    "$ref": "definitions.json#/unit"
+                },
+                "maxLimitingElementVoltage": {
+                    "description": "maximum voltage value that can be applied continuously to the resistor",
+                    "comment": "units of volts",
+                    "$ref": "definitions.json#/unit"
+                },
+                "minTemperature": {
+                    "description": "minimum temperature under which a resistor can be expected to reliably operate",
+                    "comment": "units of degrees (Celsius or Kelvin)",
+                    "$ref": "definitions.json#/unit"
+                },
+                "maxTemperature": {
+                    "description": "maximum temperature under which a resistor can be expected to reliably operate",
+                    "comment": "units of degrees (Celsius or Kelvin)",
+                    "$ref": "definitions.json#/unit"
+                },
+                "resistorDerating": {
+                    "description": "graph object to capture resistance changes with temperature",
+                    "$ref": "definitions.json#/graphDefiniton"
+                },
+                "package": {
+                    "description": "package size of resistor",
+                    "$ref": "graph.json#/package"
+                }
```

Comment:
complianceList and package properties missing

Reply:
the complianceList is part of componentID